packages/@react-native-windows/cli/src/generator-windows/index.ts

```diff
@@ -515,53 +516,45 @@ function toCppNamespace(namespace: string) {
   return namespace.replace(/\./g, '::');
 }
 
-export function installDependencies(options: {verbose: boolean}) {
-  const cwd = process.cwd();
+export async function installScriptsAndDependencies(options: {
+  verbose: boolean;
+}) {
+  const projectPackage = (await WritableNpmPackage.fromPath(
+    path.join(process.cwd(), 'package.json'),
+  ))!;
 
-  // Extract react-native peer dependency version
-  const rnwPackageJsonPath = require.resolve(
-    'react-native-windows/package.json',
-    {paths: [process.cwd()]},
-  );
-  const rnwPackageJson = JSON.parse(
-    fs.readFileSync(rnwPackageJsonPath, {encoding: 'UTF8'}),
-  );
-  let rnPeerDependency = rnwPackageJson.peerDependencies['react-native'];
```

Comment:
We don't need to do special tokenization stuff anymore. This was likely an artifact of when we depended on the microsoft/react-native fork and had a peer dependency that allowed forked or non-forked versions.